atlp-rwanda/ecommerce-app-predators-fe
# Ch configure pre commit hooks

#### What does this PR do?
Configure ESLint and Git Pre commit hook to run tests and ESlint format before commits
#### Description of Task to be completed?

> ESLint is the required linter for the project. It'll help check for style violations and flag them.
The pre-commit hook will run the following commands

format - which will be set up to use ESlint to format the code base before every commit based on ESlint and Prettier rules
test - Which will make sure that no code is committed with any test breaking changes

#### How should this be manually tested?
`git clone ch-configure-pre-commit-hooks`
`npm i`
`run npm prettier:check`
`run npm prettier:fix`
or 
`npm run lint-staged`

#### What are the relevant pivotal tracker/Trello stories?
[https://trello.com/c/pzOYtTt8/45-configure-eslint-and-git-pre-commit-hook-to-run-tests-and-eslint-format-before-commits]
(url)
#### Screenshots (if appropriate)
N/A
